src/features/posts/types/post-content.ts

```diff
@@ -1,4 +1,6 @@
 export interface PostContent {
+    postId: number;
```

Comment:
Skoro jesteś w interface PostContent to zamiast postId może być samo id, bo i tak wiadomo, że domyślnie dotyczy się ono postu. Natomiast userId jest git bo musi być tak nazwane bo jesteś w interface PostContent